Sources/MintKit/Mintfile.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  Mintfile.swift
+//  MintKit
+//
+//  Created by Wolfgang Lutz on 04.04.18.
+//
+
+import Foundation
+import PathKit
+
+struct Mintfile {
+  static let defaultPath = Path(FileManager.default.currentDirectoryPath) + "Mintfile"
+  
+  static func `default`() -> Mintfile? {
+    return self.init(path: Mintfile.defaultPath)
+  }
+  
+  let packageInfos: [PackageInfo]
```

Comment:
The struct `PackageInfo` has always had a weird name. I think I'll change it to `Package`.
Can we rename this property to `packages`.